modules/kinesis/src/it/scala/com/snowplowanalytics/snowplow/enrich/kinesis/EnrichKinesisSpec.scala

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2022-2022 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.snowplow.enrich.kinesis
+
+import scala.concurrent.duration._
+
+import cats.effect.{Blocker, IO}
+import cats.effect.concurrent.Ref
+
+import fs2.Stream
+
+import retry.syntax.all._
+import retry.RetryPolicies
+
+import cats.effect.testing.specs2.CatsIO
+
+import org.specs2.mutable.Specification
+import org.specs2.specification.AfterAll
+
+import com.snowplowanalytics.snowplow.analytics.scalasdk.Event
+
+import com.snowplowanalytics.snowplow.enrich.common.fs2.config.io.{Input, Output}
+
+class EnrichKinesisSpec extends Specification with AfterAll with CatsIO {
+
+  override protected val Timeout = 10.minutes
+
+  val retryPolicy =
+    RetryPolicies.limitRetriesByCumulativeDelay(5.minutes, RetryPolicies.capDelay[IO](2.second, RetryPolicies.fullJitter[IO](1.second)))
+
+  def afterAll: Unit = Containers.localstack.stop()
+
+  "enrich-kinesis" should {
+    "initialize with the minimal config" in {
```

Comment:
> is the config parsed correctly?

Just this. The reason why we can't test that it fully initializes properly and that it's waiting for data is that communicating with localstack requires to override the default endpoints, but we don't want to do that as it would not be running the minimal file any more. The day we can communicate with real Kinesis in the integration tests, we can go until waiting for data.